README.md

```diff
@@ -1,9 +1,23 @@
+# GRZ Tools Monorepo
+
 This monorepo hosts the following packages:
 
-- `grz-cli`
-- `grz-pydantic-models`
-- `grzctl`
-- `grz-common`
-- `grz-db`
+- [`grz-cli`](packages/grz-cli/README.md) - A command-line tool for validating, encrypting, uploading and downloading submissions to/from a GDC/GRZ (Genomrechenzentrum).
```

Comment:
I would not mention downloading for `grz-cli`, since it should only be uploading. And only to a GDC/GRZ, not from (that's grzctl)